docs/dev/guidelines/language-guidelines.rst

```diff
@@ -0,0 +1,100 @@
+***********
+Inclusive, Diversity-Sensitive, and Appreciative Language
+***********
+
+Motivation
+======
+
+Inclusive, diversity-sensitive language is important because language influences our thoughts and assessments of reality
+and actively contributes to equality.
+It is our responsibility to design software that is aware of diverse users, contributors, and developers.
+We therefore provide guidelines for inclusive, diversity-sensitive, and appreciative language used for code, documentation, and user interaction.
+
+.. note::
+    This page is not meant to be an exhaustive reference.
+    It rather describes some general guidelines and illustrates some best practices to follow.
+
+English language 
+==================================
+
+English language is used for code, documentation, user interaction.
+
+**Best practice**
+
+* **Avoid using terms that have social history**:  Terms that can have historical significance or impact in regards to race, ethnicity, national origin, gender, age, mental and physical ability, sexual orientation, socioeconomic status, religion, and educational background.
+* **Avoid using idioms and jargons**:  These can exclude people who don’t have particular specialized knowledge, and many idioms don’t translate from country to country.  Additionally, these sometimes have origins in negative stereotypes.
+* **Write inclusive examples**:  Try to avoid using examples in documentation that are culturally-specific to a particular country, and be sure to use diverse names.
```

Comment:
```suggestion
* **Write inclusive examples**:  Try to avoid using examples in the documentation that are culturally specific to a particular country, and be sure to use diverse names.
```